src/main/java/zd/zero/waifu/motivator/plugin/assets/AssetManager.kt

```diff
@@ -21,29 +22,33 @@ enum class AssetCategory(val category: String) {
     VISUAL("visuals"), AUDIBLE("audible"), TEXT("text")
 }
 
+object HttpClientFactory {
+    fun createHttpClient(): CloseableHttpClient =
```

Comment:
I needed to be able to mock the `AssetManager` and the initializations still run when you mock an object. 
So I chose to have break out the client to avoid the null pointer exception thrown from `ApplicationInfoEx.getInstance()`